charts/dependabot-gitlab/templates/servicemonitor.yaml

```diff
@@ -0,0 +1,45 @@
+{{- if and ( .Capabilities.APIVersions.Has "monitoring.coreos.com/v1" )   .Values.worker.serviceMonitor.enabled -}}
```

Comment:
I think it is better to not check for `monitoring.coreos.com/v1` and leave this decision for the user. If someone would enable serviceMonitor but api does not have `monitoring.coreos.com/v1` it would just silently be skipped and user might not even know about this.

Additionally it should also check for `.Values.metrics.enabled`